boards/arm/mimxrt1060_evk/pinmux.c

```diff
@@ -144,6 +144,25 @@ static int mimxrt1060_evk_init(const struct device *dev)
 			    IOMUXC_SW_PAD_CTL_PAD_PKE_MASK |
 			    IOMUXC_SW_PAD_CTL_PAD_SPEED(2) |
 			    IOMUXC_SW_PAD_CTL_PAD_DSE(6));
+
+#if defined (CONFIG_BT_UART_ON_DEV_NAME)
+	if (strcmp(CONFIG_BT_UART_ON_DEV_NAME, "UART_3") == 0) {
```

Comment:
This should use the dts property `hw-flow-control` instead of `CONFIG_BT_UART_ON_DEV_NAME`

Reply:
Thanks @MaureenHelm , I have upated it, please help check it again.